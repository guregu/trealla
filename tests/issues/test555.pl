:- use_module(library(iso_ext)). % for scryer
:- use_module(library(clpz)).

% if this is uncommented, it works:
% :- use_module(library(dcgs)).

:-initialization(main).

main :-
    X in 1 .. 2,
    copy_term(X, X, Attrs),
    write(Attrs), nl.

