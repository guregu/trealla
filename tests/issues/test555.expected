[clpz:(_5 in 1..2)]
